### PR TITLE
Fix EMP'ed units turning

### LINKED ISSUE #19751: TS - EMPed AttackFrontal units can turn when given attack order.
I've been having a similar issue with Magnetron in RV, decided to check TS too to see if it something on our end or if it can be reproduced in upstream. Seems like it is already the case with EMP in TS.

Since unlike EMPed units in TS, Magnetroned vehicles RV are meant to be able to fire, i think a way to prevent a unit from turning with conditions, either on Mobile: or some sort of TurnSpeedMultiplier should be able to fix this issue for both cases.

### PR DESCRIPTION
**Pull request by PunkPun**
Closes #19751

### CODE REVIEW

File: OpenRA.Mods.Common/Activities/Attack.cs [hunk 1]
```diff
@@ -212,13 +215,20 @@ protected virtual AttackStatus TickAttack(Actor self, AttackFrontal attack)
...
 			if (!attack.TargetInFiringArc(self, target, attack.Info.FacingTolerance))
 			{
-				var desiredFacing = (attack.GetTargetPosition(pos, target) - pos).Yaw;
-
-				// Don't queue a turn activity: Executing a child takes an additional tick during which the target may have moved again
-				facing.Facing = Util.TickFacing(facing.Facing, desiredFacing, facing.TurnSpeed);
-
-				// Check again if we turned enough and directly continue attacking if we did
-				if (!attack.TargetInFiringArc(self, target, attack.Info.FacingTolerance))
+				// Don't queue a turn activity: Executing a child takes an additional tick during which the target may have moved again.
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Activities/Attack.cs]: Please move this comment back to right above `TickFacing`.

**Comment by Mailaender** [comment]
Allowing units to turn gave a nice affect to #11841.

**Review by abcdefg30 — DISMISSED** [review]

**Comment by PunkPun** [comment]
I found and fixed a bug in the implementation which allowed in turning under very specific conditions

**Comment by PunkPun** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/81cc81591bec132709093013242ef2210503a5b7...182246451dbe66fc2e3f7893c9419fb3ca4ff367)

**Pull request merged**